pkg/parser/interface.go

```diff
@@ -65,6 +65,8 @@ var (
 	ErrDifferentCountMetrics = ParseError("both arguments must have the same number of metrics")
 	// ErrInvalidArgumentValue is an eval error returned when a function received an argument that has the right type but invalid value
 	ErrInvalidArgumentValue = ParseError("invalid function argument value")
+	// ErrMovingWindowSizeLessThanRetention is an eval error returned when a function received a moving window size bigger than the metrics retention
+	ErrMovingWindowSizeLessThanRetention = ParseError("moving window is bigger than the metrics retention")
```

Comment:
I'd rephrase to `windowSize param exceed metric retention`

Reply:
thanks, fixed